test/metrics_test.py

```diff
@@ -157,13 +165,14 @@ def test_emit(self):
         attr = {'pod': 'gew1'}
         tags = ['cool-metric']
 
-        mr.emit('one_time_metric', 22, attributes=attr, tags=tags)
+        mr.emit('one_time_metric', 22, attributes=attr, resources={}, tags=tags)
 
         metric = {'key': 'key',
                   'attributes': {'what': 'one_time_metric', 'pod': 'gew1'},
                   'value': 22,
                   'type': 'metric',
-                  'tags': ['cool-metric']}
+                  'tags': ['cool-metric'],
+                  'resources': {}}
```

Comment:
Similar to including a particular test for resources above, I think it would be good to include a unique-ish value here for the resources instead of an empty dict, that way a default value wouldn't satisfy this.